app-modules/authorization/src/Models/Concerns/DefinesPermissions.php

```diff
@@ -62,11 +65,13 @@ protected function webPermissions(): array
             return [];
         }
 
-        return resolve(ApplicationModules::class)
-            ->moduleConfig(
-                module: 'authorization',
-                path: 'permissions/web/model'
-            );
+        return Cache::remember('application.model.web.permissions', self::SECONDS_TO_CACHE, function () {
+            return resolve(ApplicationModules::class)
+                ->moduleConfig(
+                    module: 'authorization',
+                    path: 'permissions/web/model'
+                );
+        });
```

Comment:
If we deployed changes where permissions would be different less than an hour in between would there be an issue here?

Do we need some sort of cache bust option during seeding?

Reply:
I think we can just reduce the caching time here. Even though the model web/api permissions are likely unchanging, we can read the file once per seeding event and cache for max approximate life of the seeding itself. Or, we can just bust the cache after seeding has completed. Both routes potentially make sense, let me know which you prefer.

Reply:
Let's bust the cache once seeding is completed